HealthMonitoring.AcceptanceTests/HealthMonitoring.AcceptanceTests.csproj

```diff
@@ -68,6 +68,7 @@
     <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
     <Reference Include="System.Data" />
+    <Reference Include="System.Drawing" />
```

Comment:
No idea. There are a few others within the project as well that have no uses. It was selected during nuget package update